docs/conf.py

```diff
@@ -82,9 +82,12 @@
 autodoc_preserve_defaults = True
 autodoc_member_order = "groupwise"
 
-suppress_warning = []
+# the open documentation links to the enterprise documentation in some places
+# let those links be broken in the open-only documentation and don't complain
+suppress_warnings = ["ref.ref"]
```

Comment:
does this mean we don't get this check in enterprise?

Reply:
I _believe_ that the enterprise build has its own `conf.py` and symlinks to all of the open docs files. So the enterprise build should not be suppressing this warning.

Reply:
Then this seems fine to me; in general I'm not sure how to feel about broken links on our public docs site.

Reply:
The link that I added this for is hidden behind an `only:: internal`. So for this specific use case we only have a broken link in internal documentation that doesn't include the enterprise docs, which I'm not super worried about. I'm worried about ignoring warnings at this granularity - I would have preferred to suppress the warning for a single broken link.

Reply:
OK I guess ci on enterprise would detect broken links in the open repo anyway? (Trying to figure out how much of this cake we have that we get to eat)

Reply:
Yeah, I think a truly broken link would get caught in enterprise CI, which is better than nothing. I think our other options are 

1. Don't link from open repo docs to enterprise repo docs - which makes addenda-style enterprise docs less useful.
2. Keep trying to find a more granular way to ignore broken links - which I think could be expensive in developer time given how much time I have already dedicated to this.

Reply:
thanks; this is fine with me but I don't feel like I have enough context/authority over docs to be final approval here.

Reply:
Sorry, this is not okay. Suppressing this warning would result in broken links not being caught by testing and potentially making its way into public-facing documentation